apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -285,8 +275,6 @@ const EventTypePage = (props: EventTypeSetupProps) => {
             locations,
             metadata,
             customInputs,
-            hosts: hostsInput,
-            hostsFixed,
             // We don't need to send send these values to the backend
             // eslint-disable-next-line @typescript-eslint/no-unused-vars
             seatsPerTimeSlotEnabled,
```

Comment:
no used?

Reply:
Not anymore, hosts no longer needs any changing after load so it comes in through ...input later on - a bit simpler than before